pycbc/workflow/jobsetup.py

```diff
@@ -1021,8 +1021,9 @@ def create_node(self, bank, tags=None):
 
         # Get the output (taken from inspiral.py)
         out_files = FileList([])
+        n_dp = math.ceil(math.log10(self.num_banks))
         for i in range( 0, self.num_banks):
-            curr_tag = 'bank%d' %(i)
```

Comment:
@GarethCabournDavies I'm worried about including this change on v2.3.10 as it breaks the ability to reuse files (in a bad way, as pegasus gets confused by reusing a bunch of random files in a workflow which isn't well mapped to the workflow graph and so takes over a day to plan). Will it break anything if I revert this?

Reply:
Line 347 of `tmpltbank.py` is linked and so that would need changed as well.

I don't fully understand how this is breaking the file reuse though? All this is doing is adding zero padding to the number

Reply:
It means I can't reuse files made in v2.3.9 (or earlier) when running workflows created with this patch in place.

Reply:
Okay, that's fair